packages/forms/src/templates/ArrayFieldTemplate.scss

```diff
@@ -50,26 +72,39 @@ $tf-filterwidget-shadow-color: $shadow !default;
 		.element {
 			flex-grow: 1;
 			padding: 0;
+			overflow: hidden;
 
 			> div {
 				padding: 0;
 				margin: 0;
 			}
 
 			fieldset {
-				div {
-					border-bottom: 1px solid $gray-lighter;
+				> div:first-child {
+					border-bottom: 1px solid $wild-sand;
+					// to forbid children selection of text on double click
+					user-select: none;
+					-moz-user-select: none;
+					-ms-user-select: none;
```

Comment:
please remove this and see why postcss loader is not doing it's job.
Examples at http://autoprefixer.github.io/ show it should work